### ISSUE TITLE
Adding null (empty) alt text to superfluous images in posts / making images in posts aria-hidden

### ISSUE THREAD

**Issue by cygaj** [bluesky-social/social-app#3345]
_**Disclaimer:** I have not tested what I'm describing using a screen reader. All my understanding of alt texts come from guides like ones on W3C, like [this one](https://www.w3.org/WAI/tutorials/images/decorative/). I don't know anyone who uses a screen reader.
I am also not a coder. I'm only here on github as a lay person wishing to share feedback._

**Problem**
Sometimes posts that are text-based have images added to them for ambience, decoration, to grab attention of a sighted user, or exemplify what is already described sufficiently in the post itself. I choose to skip alt texts in such situations but as far as I understand a screen reader would still announce that there's an image present, which is undesirable.

**Solution**
Allow users to set alt text to "null". Include a succinct explanation in the UI when the option should be used.

**Alternatives**
Adding an alt text to every single image and creating clutter.

_____________

**Related**
I have no idea how screen readers interpret elements of bluesky but would it be possible to hide from them entire posts that _solely_ contain alt-text-less images? I am concerned about the culture of scolding people about not adding alt texts to images that have… no real value besides visual. But cluttering feed with empty content for users of screen readers seems like _the_ problem to me and I feel maybe the solution could be technical?
**EDIT:** alternatively I would love some sort of option to self-label a post as "purely visual content" and have an option in moderation to hide all such from the feed
**EDIT:** ↳[Add a native setting to hide from the feed posts containing images without alt text #4550](https://github.com/bluesky-social/social-app/issues/4550)
